pkgs/by-name/sq/sqlline/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  maven,
+  fetchFromGitHub,
+  makeWrapper,
+  jre,
+  docbook_xml_dtd_42,
+}:
+
+maven.buildMavenPackage rec {
+  pname = "sqlline";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner = "julianhyde";
+    repo = "sqlline";
+    rev = "sqlline-${version}";
+    hash = "sha256-rUlGtMgTfhciQVif0KaUcuY28wh+PrHsKen8qODom24=";
+  };
+
+  mvnHash = "sha256-9qDzc6TRn9Yv3/nTATZgP6J+PTZEZCN1et/3GrRb7X4=";
+
+  nativeBuildInputs = [
+    makeWrapper
+    docbook_xml_dtd_42
+  ];
+
+  mvnParameters = "-DskipTests";
+
+  # Patch the DOCTYPE declaration in manual.xml
+  postPatch = ''
+    sed -i 's|https://docbook.org/xml/4.2/docbookx.dtd|${docbook_xml_dtd_42}/xml/dtd/docbook/docbookx.dtd|' src/docbkx/manual.xml
+    sed -i 's|PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN"|PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"|' src/docbkx/manual.xml
+  '';
+
+  buildOffline = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/java $out/bin
+    cp target/sqlline-${version}-jar-with-dependencies.jar $out/share/java/sqlline-${version}.jar
+    makeWrapper ${jre}/bin/java $out/bin/sqlline \
+      --add-flags "-jar $out/share/java/sqlline-${version}.jar"
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Shell for issuing SQL to relational databases via JDBC";
+    homepage = "https://github.com/julianhyde/sqlline";
+    mainProgram = "sqlline";
+    license = lib.licenses.bsd3;
+    platforms = lib.platforms.all;
```

Would probably better if it's all platforms that Java supports - then again, I wonder if  buildMavenPackage already sets this for you (?)
```suggestion
	inherit (jre.meta) platforms;
```

---

It does indeed, I will delete this tag completely and let `buildMavenPackage` handle it